common/src/main/java/whocraft/tardis_refined/registry/SoundRegistry.java

```diff
@@ -11,6 +11,7 @@ public class SoundRegistry {
     public static final RegistrySupplier<SoundEvent> TARDIS_TAKEOFF = setUpSound("tardis_takeoff");
     public static final RegistrySupplier<SoundEvent> TARDIS_LAND = setUpSound("tardis_land");
     public static final RegistrySupplier<SoundEvent> TARDIS_SINGLE_FLY = setUpSound("tardis_single_fly");
+    public static final RegistrySupplier<SoundEvent> PATTERN_MANIPULATOR = setUpSound("pattern_manipulator");
```

Comment:
There's been a big conscience effort to not use any sounds other than flights, however, the sound is good so I'll allow it. Just something for future note.